dto/src/main/java/ca/on/oicr/gsi/runscanner/dto/dragen/samplesheet/Samplesheet.java

```diff
@@ -0,0 +1,42 @@
+package ca.on.oicr.gsi.runscanner.dto.dragen.samplesheet;
+
+import java.time.Instant;
+import java.util.LinkedList;
+import java.util.List;
+
+public class Samplesheet {
+  private List<SamplesheetSection> info;
+  private Instant mtime;
```

Comment:
Please use similar naming for all these time fields.

- `AnalysisFile` has `created` and `modified`
- `DragenWorkflowAnalysis` has `startTime` and `completionTime`
- Here is `mtime`

Reply:
Made these a bit more consistent.